tests/checkers/unittest_refactoring.py

```diff
@@ -16,7 +16,10 @@
 @pytest.mark.timeout(8)
 def test_process_tokens() -> None:
     with pytest.raises(SystemExit) as cm:
-        Run([os.path.join(REGR_DATA, "very_long_line.py")], reporter=TextReporter())
+        Run(
+            [os.path.join(REGR_DATA, "very_long_line.py"), "--disable=C"],
```

Comment:
Do we disable `C` in our own `pylintrc`?

Reply:
No but some like ``invalid-name`` and ``missing-docstring`` that happens very often.